docs/manual/troubleshooting.tex

```diff
@@ -1038,6 +1038,15 @@ \subsection{Build the Checker Framework Manual (this document)\label{building-ma
 \end{enumerate}
 
 
+\subsection{Configure Eclipse to edit the Checker Framework\label{building-eclipse}}
+\begin{enumerate}
+\item Download Eclipse from the \href{https://www.eclipse.org/downloads}{official Eclipse website} and install it.
+\item Once the installation is completed, run Eclipse and set the workspace to \$JSR308.
+\item Enter the main Eclipse working screen and under "File" tab, select Import, General, and then Existing Projects into workspace. After the Import Projects window appears, select "Select Root Directory", browse for \$JSR308 folder, and select the whole folder. Try to deselect the projects related to Eclipse, encryption, tutorial, and demo.
+\item After all projects have been imported, everything must be built successfully.
```

Comment:
Could you clarify this?  Does "must be" mean "the user should do something", or "Eclipse is guaranteed to have done something", or does it mean something else?